bazel/external_crates.bzl

```diff
@@ -234,6 +234,12 @@ def external_crates_repository(name, cargo_lockfile, lockfile, sanitizers_enable
             "bit-vec": crate.spec(
                 version = "^0.6.3",
             ),
+            "bitcoin-0-32": crate.spec(
+                package = "bitcoin",
+                git = "https://github.com/ninegua/rust-bitcoin",
```

Comment:
Is there any hope of having those changes merged up stream? If we really need to fork `bitcoin`, then I guess it should be in a repo owned by Dfinity and not your own for maintainability reasons.